src/test/java/com/artipie/asto/S3StorageTest.java

```diff
@@ -68,6 +71,17 @@ void shouldUploadObjectWhenSave(final AmazonS3 client) throws Exception {
         MatcherAssert.assertThat(downloaded, Matchers.equalTo(data));
     }
 
+    @Test
+    void shouldGetObjectWhenLoad(final AmazonS3 client) {
+        final String bucket = UUID.randomUUID().toString();
+        client.createBucket(bucket);
+        final byte[] data = "data".getBytes();
+        final String key = "some/key";
+        client.putObject(bucket, key, new ByteArrayInputStream(data), new ObjectMetadata());
+        final byte[] value = new BlockingStorage(this.storage(bucket)).value(new Key.From(key));
+        MatcherAssert.assertThat(value, Matchers.equalTo(data));
```

Comment:
@olegmoz Please add assertion failure message and do not use static matchers. You can also inline value variable into the assertion.